Dockerfile_ubi

```diff
@@ -0,0 +1,33 @@
+ARG BUILD_TAG
+ARG REPO_URL=sumologic-otel-collector
+ARG BUILD_ARCH
+
+## Build RedHat compliant image
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.9
+
+ENV SUMMARY="UBI based sumologic-otel-collector" \
+    DESCRIPTION="Sumo Logic Distribution for OpenTelemetry Collector is a Sumo Logic-supported distribution of the OpenTelemetry Collector. It is a single agent to send logs, metrics and traces to Sumo Logic."
+
+LABEL name="sumologic-otel-collector" \
+      vendor="Sumo Logic" \
+      version="$BUILD_TAG" \
+      release="4" \
```

Comment:
What is this label for?

Reply:
I wish I knew! This was already in the original Dockerfile for ubi, so I left it: https://github.com/SumoLogic/sumologic-openshift-images/blob/main/sumologic-otel-collector/0.57.2-sumo-1/Dockerfile

Reply:
@kkujawa-sumo do you know?

Reply:
Looking at this: https://developers.redhat.com/articles/2021/11/11/best-practices-building-images-pass-red-hat-container-certification#best_practice__6__embed_identifying_information_inside_your_image, this should probably just be `0`.

We should probably also apply security updates as per section 5 of that guide.

Reply:
Doesn't `microdnf update` already apply these updates? The command in the section 5 does not work with `microdnf` (some unsupported flags).

Reply:
Also, what should be equal to `0`? The `release` label?